app/views/components/_actions-group.html.erb

```diff
@@ -11,9 +11,13 @@
       <% subsections.each do |subsection| %>
         <div data-ec-list-subsection="<%= subsection[:title] %>">
           <h3 class="govuk-heading-m"><%= subsection[:title] %></h3>
-          <% if subsection[:items].any? %>
-            <ul class="govuk-list">
-              <% subsection[:items].each do |item| %>
+          <% %i[items support_and_advice_items].each do |item_type| %>
+            <% if subsection[item_type].any? %>
+              <% if item_type == :support_items %>
+                <h3 class="govuk-heading-s"><%= t("coronavirus_form.results.support_and_advice_text") %></h3>
```

Comment:
At first glance I thought the the `h3` on line 17 should be an `h4` since visually this heading appears to be nested underneath the subsection title on line 13.

But looking at the Trello card it could semantically make sense to be either a `h3` or a `h4`.

@alex-ju - what do you think?

Reply:
I'm getting an `undefined method `any?' for nil:NilClass` error while trying to preview the component (e.g. `http://localhost:5000/component-guide/actions-group`), we may need to initialise `subsection ` as an array or so. I'll go through the Trello cards to try and understand the requirements.

Reply:
Thanks for fixing this @steveholmes. To me, it seems like an `h4` and since the component makes sure that an `h3` exists and we're not skipping levels I would suggest we go with that.